internal/tools/aws/database.go

```diff
@@ -183,7 +184,12 @@ func (d *RDSDatabase) rdsDatabaseProvision(installationID string, logger log.Fie
 		return err
 	}
 
-	err = d.client.rdsEnsureDBClusterCreated(awsID, *vpcs[0].VpcId, rdsSecret.MasterUsername, rdsSecret.MasterPassword, logger)
+	encryptionKey, err := d.client.kmsCreateSymmetricKey(awsID, KMSAliasNameRDS(awsID), "Key used for encrypting RDS database")
```

Comment:
What happens if the key is created on an attempt and then an error happens later? When the provisioner runs this provision flow again, will is return an error or will the encryption key be returned on the already-created key?

Reply:
No, even if [CreateAlias](https://github.com/mattermost/mattermost-cloud/pull/182/files#diff-d5ff833767547cbc731fa146b4d30688R20) fails it will still be able to re-run. However, if creating alias failed we should schedule deletion for the created key.